.github/workflows/memory-leak-testing.yml

```diff
@@ -0,0 +1,47 @@
+name: memory leak testing
+
+on:
+  pull_request:
+    branches: [ "main" ]
+
+jobs:
+  memory-leak-testing:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: ${{ vars.NODE_VERSION }}
+
+      - name: Cache pnpm dependencies
+        id: cache-pnpm-dependencies
+        uses: actions/cache@v3
+        with:
+          path: node_modules
+          key: ${{ runner.os }}-dependencies-${{ hashFiles('**/pnpm-lock.yaml') }}
+          restore-keys: |
+            ${{ runner.os }}-dependencies-
+
+      - name: Install pnpm
+        run: npm install -g pnpm
+
+      - name: Install memlab
+        run: npm install -g memlab
+
+      - name: Install dependencies
+        run: pnpm i --frozen-lockfile
+        if: |
+          steps.cache-pnpm-dependencies.outputs.cache-hit != 'true' ||
+          steps.cache-pnpm-packages.outputs.cache-hit != 'true'
+
+      - name: Run dev
+        run: pnpm run dev &
```

Comment:
I think that it will be good to remove unnecessary & from the end of line
What do you think ?

Reply:
with this & we have a possibility to run the dev server without blocking memlab test, if we remove it, dev server will be stated and will not be ended